internal/querycoord/impl.go

```diff
@@ -89,16 +89,18 @@ func (qc *QueryCoord) GetStatisticsChannel(ctx context.Context) (*milvuspb.Strin
 
 // ShowCollections return all the collections that have been loaded
 func (qc *QueryCoord) ShowCollections(ctx context.Context, req *querypb.ShowCollectionsRequest) (*querypb.ShowCollectionsResponse, error) {
-	dbID := req.DbID
-	log.Debug("show collection start", zap.Int64("dbID", dbID))
+	log.Debug("show collection start",
+		zap.String("role", typeutil.QueryCoordRole),
+		zap.Int64s("collectionIDs", req.CollectionIDs),
+		zap.Int64("msgID", req.Base.MsgID))
 	status := &commonpb.Status{
 		ErrorCode: commonpb.ErrorCode_Success,
 	}
 	if qc.stateCode.Load() != internalpb.StateCode_Healthy {
 		status.ErrorCode = commonpb.ErrorCode_UnexpectedError
-		err := errors.New("query coordinator is not healthy")
+		err := errors.New("QueryCoord is not healthy")
 		status.Reason = err.Error()
-		log.Debug("show collection end with query coordinator not healthy")
+		log.Warn("show collection failed", zap.String("role", typeutil.QueryCoordRole), zap.Int64("msgID", req.Base.MsgID), zap.Error(err))
 		return &querypb.ShowCollectionsResponse{
 			Status: status,
 		}, nil
```

Comment:
use err as a return value

Reply:
Modifying common proto requires too much code to change in  all components, and if it is really necessary, it is recommended to make a separate pr.

Reply:
error is only used to receive grpc information